src/execute-script.html

```diff
@@ -0,0 +1,54 @@
+<script type="text/javascript">
+
+    RED.nodes.registerType('execute-script', {
+        category: 'Webdriver IO',
+        color: '#a6bbcf',
+        defaults: {
+            name: {value: ''},
+            locateUsing: {value: 'xpath'},
+            locateValue: {value: ''},
+            action: {value: 'sync'},
+            script: {value: ''}
+        },
+        inputs: 1,
+        outputs: 1,
+        icon: 'white-globe.png',
+        label: function() {
+            return this.name || "execute script"
+        }
+    })
+</script>
+
```

Comment:
Why does the execute script node need to locate an element? Doesn't it just execute javascript in the browser?

Reply:
It does! But it also find the element and pass element object as argument. which was java/C# selenium pattern!